tests/Endpoints/ProgramYearObjectiveTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Endpoints;
+
+use App\Tests\DataLoader\ObjectiveData;
+use App\Tests\DataLoader\ProgramYearData;
+use App\Tests\DataLoader\TermData;
+use App\Tests\ReadWriteEndpointTest;
+
+/**
+ * ProgramYearObjectiveTest API endpoint Test.
+ * @group api_1
```

Comment:
All the new XObjectiveTests shouldn't be in the same group.

Reply:
done. i spread those out over api test groups 1-3.